updated screenshot gitbook page to version 2307.0 (#30)
iXtenda/iVP-gitbook: home/ivp-planning/advanced-tools/creating-screenshots.md

# Creating Screenshots
  

> It is possible to save the current content of the 3D- or 2D-Panel as a screenshot. To do so, first click the Screenshot-Button in either panel.  

Reviewer: ```suggestion
It is possible to save the current content of the 3D- or 2D-Panel as a screenshot. To do so, first click the screenshot button in either panel.  
```

> A new window will open, in which you will be prompted to name your screenshot.

Reviewer: ```suggestion
A new window will open, in which you will be prompted to pick a location for your screenshot.
```

![](../../../.gitbook/assets/screenshot-creation.png)  

## Changing the Screenshot Resolution

> Before saving your screenshot, you have the option to change your screenshots resolution. Options range from 1280x720 to 7680x4320.

Reviewer: We need to rephrase this to not include the image width here, as it depends on the aspect ratio.

![](../../../.gitbook/assets/screenshot-resolution.png)

> ## Marker- and Eraser-Tools

Reviewer: I would prefer to move this section to the bottom because changing aspect ratio & setting transparency is no longer possible once you're in the annotation mode.

> Before saving your sreenshot, you also have the option to use the marker-tool to highlight any given object inside the 3D- or 2D-Panel. To do so, click on the "Annotate" button highlighted below.  

Reviewer: ```suggestion
Before saving your sreenshot, you also have the option to use the annotation tool to highlight any given object inside the 3D- or 2D-Panel. To do so, click on the "Annotate" button highlighted below.  
```

To erase any unwanted markings, you can use either the Eraser- or Clear-Tools.

![](../../../.gitbook/assets/screenshot-marker-eraser-tool.png)

## Changing the Aspect Ratio of your Screenshot

You can change the aspect ratio of your screenshot by clicking on the "Aspect ratio" button, to reveal a dropdown menu.  

![](../../../.gitbook/assets/screenshot-aspectratio.png)

## Transparent Background

> You have the option to create a screenshot without any background noises, by clicking on the "Transparent Background" option.  

Reviewer: ```suggestion
You have the option to create a screenshot without any background, by clicking on the "Transparent Background" option.  
```
